Entities/Enums/ExchangeStatus.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace FRITeam.Swapify.Entities.Enums
+{
+    public enum ExchangeStatus
+    {
+        WaitingForExchange = 0,
+        Done = 1
```

Comment:
I think there is cancelled status missing

Reply:
For now it is not needed